### ISSUE TITLE
Code highlighting in the preview panel

### ISSUE THREAD

**Issue by y1zhou** [Requarks/wiki#1162]
**Describe the bug**
I think this was also mentioned in #1055. When editing a page, the code panels in the preview panel are not highlighted.

**To Reproduce**
1. Edit any page
2. Add a code block with a specified language, e.g. `bash`
3. Check out the preview panel

**Expected behavior**
Code should be highlighted in the preview panel.

**Host Info (please complete the following information):**
 - OS: Ubuntu 16.04 xenial, Docker 18.06.0-ce
 - Wiki.js version: 2.0.0-rc.17
 - Database engine: postgres 12-alphine docker image

**Additional context**
According to the error message in the console, it's trying to fetch `/e/en/page/components/prism-bash.min.js` instead of `/js/prism/prism-bash.min.js`.

**Comment by kesslerdev** [comment]
same with 2.0.1 but also in "view" page

```
GET https://*****/js/prism/prism-golang.min.js net::ERR_ABORTED 404 (Not Found)
```
deployed on heroku

**Comment by NGPixel** [comment]
Fixed by f154f9cae5d9e6d0f2012e0040e87db96d6e2537
